// This file is part of DynExp.

syntax = "proto3";

import public "Common.proto";

package DynExpProto.NetworkDigitalOut;

message SampleMessage
{
	bool Value = 1;
}

service NetworkDigitalOut
{
	rpc SetAsync (SampleMessage) returns (DynExpProto.Common.VoidMessage) {}
	rpc SetSync (SampleMessage) returns (DynExpProto.Common.VoidMessage) {}
	rpc SetDefault (DynExpProto.Common.VoidMessage) returns (DynExpProto.Common.VoidMessage) {}
}